automatic/360ts/360ts.nuspec

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
+  <metadata>
+    <id>360ts</id>
+    <version>9.0.0.1146</version>
+    <title>360 Total Security</title>
+    <authors>Qihu 360 Software</authors>
+    <owners>Chocolatey</owners>
+    <licenseUrl>http://www.360totalsecurity.com/en/about.html</licenseUrl>
+    <projectUrl>http://360safe.com/</projectUrl>
+    <iconUrl> https://cdn.rawgit.com/chocolatey/chocolatey-coreteampackages/b92d42be38598e16e7e526b3b8b43c3220439cd4/icons/360ts.png</iconUrl>
+    <requireLicenseAcceptance>false</requireLicenseAcceptance>
+    <description>
+      A feature-packed software solution that provides users with a powerful antivirus, a junk cleaner and a system booster within the same interface
+    
+     ## Features
+      
+      - 360 Total Security offers your PC complete protection from Viruses, Trojans and other emerging threats.
+      - Whether you are shopping online, downloading files or chatting with your friends you can be sure that 360 Total Security is there to keep you safe and your computer optimized.
+      - Clean-up utility is just one click away to keep your PC in optimal condition.
+    
+    ## Notes
+       
+      - Full Check: Quickly check the overall condition of your computers' health and safety with one click.
+      - Virus Scan: Integrating award winning antivirus engines from 360 Cloud Scan Engine,360 QVMII AI Engine, Avira and Bitdefender to provide you with the ultimate in Virus detection and protection capabilities.
+      - Speedup: Manage and optimize your system services, boot up items and plugins - Shorten your boot time and get going sooner!
+      - Cleanup: Cleanup frees your disk space by removing junk files and plugin, which can improve system performance - With cleanup you can decide which areas and files to clean.
+      - Protection: Protection offers 4 different user selectable modes - Performance/Balanced/Security and Custom. Each mode offers a different level of protection from malware, phishing attacks and backdoors.
+      
+      - Top-rated tools: 
+      - WiFi Security Check
+      - Sandbox
+      - Patch Up
+    </description>
+    <summary>A feature-packed software solution that provides users with a powerful antivirus, a junk cleaner and a system booster within the same interface</summary>
+    <copyright />
```

Comment:
remove empty elements that isn't needed.
But preferred is to add the copyright message (it is located at the bottom of the software site, ie: ` © 2014 - 2017 QIHU 360 SOFTWARE CO. LIMITED`)